ruby_on_rails/aws.md

```diff
@@ -0,0 +1,60 @@
+# Amazon Services
+
+## Amazon S3
+
+S3 is Amazon's Simple Cloud Storage Service‎, and used in most of your projects to store images and
+files.
+
+## Amazon CloudFront
+
+CloudFront is a large scale, global, and feature rich CDN. We mostly use it together with S3, since fetching data from S3 is only
+possible together with CloudFront. You could also host a Single Page Application (SPA), but for that, we most often use Google Firebase.
+
+## Setup
+
+### Preconditions
+
+You will need Renuo-CLI to be set up and at the newest version:
+
+`gem install renuo-cli` --> see [renuo-cli](https://github.com/renuo/renuo-cli)
+
+Make sure `renuo -v` shows the [newest version](https://github.com/renuo/renuo-cli/tags)
+
+### Start the Setup
+
+The following command will guide you through a setup to be able to run all the commands, you
```

Comment:
can you rephrase? I cannot understand the meaning of this sentence

Reply:
what about 
```md
The following command will create commands to set up S3 and CloudFlare. Before that, the command makes sure, you have the aws-profile and the aws-cli set up, so you have the necessary rights to run the commands.
```

Reply:
better, now I got it :)